src/game/AI/ScriptDevAI/scripts/kalimdor/dire_maul/dire_maul.cpp

```diff
@@ -308,6 +308,18 @@ struct RitualCandleAura : public SpellScript
     }
 };
 
+// 22876 - Summon Netherwalker
+struct SummonNetherWalker : public SpellScript
+{
+    void OnSuccessfulFinish(Spell* spell) const override
+    {
+        Creature* caster = static_cast<Creature*>(spell->GetCaster());
+        if (!caster || !caster->IsAlive() || !caster->IsCreature())
+            return;
+        caster->ForcedDespawn();
```

Comment:
1000 ms delay, make mob passive and set combat script to true cos its not reacting in that 1 sec delay

Reply:
From sniff btw

Reply:
will do